backend/src/main/java/pl/project/plannerapp/controller/DietController.java

```diff
@@ -43,8 +43,9 @@ public DietDTO getDietById(@PathVariable Long dietId) {
 
     @Transactional
     @PostMapping("/{dietId}")
-    public void addDiet(@PathVariable Long dietId, @RequestBody DietDTO dietJson) {
-
+    public Long addDiet(@PathVariable Long dietId, @RequestBody DietDTO dietJson) {
+        dietService.addDiet(DietConventerUtils.convert(dietJson));
+        return dietId;
```

Comment:
good job! what can happen if we dare to save a new diet with an `id` that already exists in the database? :)